src/component/handlers/edit/setClipboardData.js

```diff
@@ -12,7 +12,38 @@
 
 'use strict';
 
-function setClipboardData(e: SyntheticClipboardEvent, { text, html }): void {
+import type DraftEditor from 'DraftEditor.react';
+
+var UserAgent = require('UserAgent');
+var setImmediate = require('setImmediate');
+
+var isIE = UserAgent.isBrowser('IE');
+
+function setClipboardData(e: SyntheticClipboardEvent, editor: DraftEditor, { text, html }): void {
+  if (isIE) {
```

Comment:
This should do the same setMode thing to prevent draft from knowing about this sleight of hand.